superset-frontend/src/dashboard/components/menu/WithPopoverMenu.jsx

```diff
@@ -38,7 +38,9 @@ const defaultProps = {
   menuItems: [],
   isFocused: false,
   shouldFocus: (event, container) =>
-    container && container.contains(event.target),
+    (container && container.contains(event.target)) ||
```

Comment:
Could use the null coalescing operator here, 
```suggestion
    (container?.contains(event.target)) ||
```

Reply:
Great suggestion! Done.